package-lock.json

```diff
@@ -12727,22 +12739,20 @@
       }
     },
     "node_modules/gdal-async": {
-      "version": "3.7.0",
-      "resolved": "https://registry.npmjs.org/gdal-async/-/gdal-async-3.7.0.tgz",
-      "integrity": "sha512-UeG7RPw3xDJa759QIcZ5LM0OhsIqqDZo39WSCYZfve53oC9wjjdUD+qmXauB9dpbDCYE+MGKO4QKCVDzX/FaDQ==",
+      "version": "3.5.0",
```

Comment:
Here's the change back to 3.5.0

Reply:
This change should be made in package.json. We don't want to edit the package-lock.json directly.